addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.CopyClipboardField" owl="1">
```

Comment:
@SimonGenin Is there a reason for the copy button to have the `btn-sm` class? It forces us to set the class on the close button too, and it looks weird if we do, compared to other footers in Odoo.

![image](https://user-images.githubusercontent.com/15896813/197003738-cb3679cd-1010-4984-b6a2-05c11b220f60.png)

Reply:
You can remove it from the CopyButton component template, but use the className props so you can only add btn-sm when needed.

For the case of CopyCliboardButton field widget, you can override the getter to remove this classname from the main list of props, while keeping it in the case of CopyClipboardURL/Char/Text fields

Reply:
Is that ok for you now @Iucapad?